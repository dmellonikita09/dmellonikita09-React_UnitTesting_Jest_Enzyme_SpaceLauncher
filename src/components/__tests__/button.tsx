import { mount } from 'enzyme';
import React from 'react';
import Button from '../button';

describe('Button', () => {
  it('renders without error', () => {
    const wrapper = mount(<Button>Hello World </Button>);
    wrapper.simulate('click');
  });

  it('matches snapshot', () => {
    const component = mount(<Button>Hello World </Button>);
    expect(component).toMatchSnapshot();
  });
});
