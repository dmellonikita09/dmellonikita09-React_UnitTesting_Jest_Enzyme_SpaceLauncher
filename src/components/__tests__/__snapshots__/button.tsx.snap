// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
<Styled(button)>
  <button
    className="css-wwcn44"
  >
    Hello World 
  </button>
</Styled(button)>
`;
